src/dashboard-button/dashboard-button.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./dashboard-button.component.css";
+
+export default function DashboardButton(props: DashboardButtonProps) {
+  return (
+    <a
+      className={`omrs-link omrs-filled-neutral ${styles.link}`}
+      onClick={event => navigate(event, props, props.link)}
```

Comment:
If these are going to link within the SPA, we should be using <Link> instead of <a>. It may be that you need to cover both cases depending on where the config is directing the user.

Reply:
Sorry, I don't understand? Yes, we do need to cover both within-spa and external links.

Reply:
I think this is fine for now. We can discuss when thinks links go to places inside the app.